config/settings.yml

```diff
@@ -72,3 +72,7 @@ discourse_post_event:
     list_type: simple
     client: true
     default: ""
+  events_calendar_category:
```

Comment:
Small thing I noticed here, we should rename this to `events_calendar_categories` because this allows multiple selections (which is good, it's nice that multiple categories can each have their calendar.)